developer_manual/core/ocs-share-api.rst

```diff
@@ -255,12 +255,12 @@ shareType    int     The type of the share. This can be one of: 0 = user,
 shareWith    string  The user or group id with which the file should be shared.
 publicUpload boolean Whether to allow public upload to a public shared folder.
 password     string  The password to protect public link share with.
-permissions  int     The permissions to set on the share. 
-                     1 = read; 2 = update; 4 = create; 8 = delete;
-                     16 = share; 31 = all (default: 31, for public shares: 1)
-expireDate   string  An expire date for public link shares. 
-                     This argument expects a date string in the following 
-                     format ``'YYYY-MM-DD'``.
+permissions  int     The permissions to set on the share.
+                     1 = read (default for public shares); 2 = update;
+                     4 = create; 8 = delete; 16 = share.
+expireDate   string  An expire date for public link shares.
+                     This argument expects a date string in the following
+                     format ``'YYYY=MM=DD'``.
```

Comment:
This is suprising. Do the year, month,day fields really need "=" between them?

Reply:
Hmmm, it's been ages since I've looked at this. I'll check again.

Reply:
You're right, it is very strange. Not sure why I didn't pick that up earlier. Double-checking https://github.com/owncloud/core/blob/master/apps/files_sharing/lib/API/Share20OCS.php#L841, the correct format is `YYYY-MM-DD`.